### DISCUSSION TITLE
Seed To Different Wallets

### DISCUSSION THREAD

**Discussion by MarkFlyer** [1200wd/bitcoinlib#341] [Q&A]
Is it possible to use Bitcoin lib to extract **Litecoin**, **Ethereum**, and **USDT** Wallet Address from a mnemonic seed phrase?

**Comment by mccwdev** [comment]
Yes you can use a mnemonic seed phrase to create a Litecoin or Bitcoin wallet with bitcoinlib.

``` python
passphrase = "lumber model sunny service license typical runway clean shuffle anchor fetch adjust"
w = Wallet.create("wallet_passphrase_import", keys=passphrase, network='bitcoin')
account_ltc1 = w.new_account('Account LTC', network='litecoin')
w.get_key()
w.info()

```

USDT and Ethereum are not supported by bitcoinlib, but I assume they use the same BIP39 mnenomic word list.

**Reply by MarkFlyer** [reply]
I used this code when I create a wallet, and checked the balance it shows 0, but I already have 17$ in my LTC account
